app/src/main/java/com/lftechnology/unito/fragment/ScreenSlideTopFragment.java

```diff
@@ -0,0 +1,198 @@
+ */
+public class ScreenSlideTopFragment extends BaseFragment {
+
+    private static final String POSITION = "position";
+    private static final String DATASET = "dataset";
+    private static final String SELECTED_CONVERSION = "selectedConversion";
+
+    private int mVisibleFragmentPosition;
+    private int mPosition;
+    private String[] mDataset;
+    private ViewGroup mRootView;
+    private CustomEditText mFromUnit;
+    private String mSelectedConversion;
+
+    public ScreenSlideTopFragment() {
+    }
+
+    public static ScreenSlideTopFragment newInstance(int position, String[] dataset, String selectedConversion) {
+        ScreenSlideTopFragment fragment = new ScreenSlideTopFragment();
+        Bundle args = new Bundle();
+        args.putInt(POSITION, position);
+        args.putStringArray(DATASET, dataset);
+        args.putString(SELECTED_CONVERSION, selectedConversion);
+        fragment.setArguments(args);
+        return fragment;
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        if (getArguments() != null) {
+            mPosition = getArguments().getInt(POSITION);
+            mDataset = getArguments().getStringArray(DATASET);
+            mSelectedConversion = getArguments().getString(SELECTED_CONVERSION);
+        }
+    }
+
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        mRootView = (ViewGroup) inflater.inflate(R.layout.fragment_slider_top, container, false);
+
+        View gestureView = mRootView.findViewById(R.id.content_top);
+        gestureView.setClickable(true);
+        gestureView.setFocusable(true);
+        GestureDetector.SimpleOnGestureListener gestureListener = new GestureListener();
+        final GestureDetector gd = new GestureDetector(getActivity(), gestureListener);
+        assert gestureView != null;
+        gestureView.setOnTouchListener(new View.OnTouchListener() {
+            @Override
+            public boolean onTouch(View view, MotionEvent motionEvent) {
+                gd.onTouchEvent(motionEvent);
+                return false;
+            }
+        });
+
+        return mRootView;
+    }
+
+    @Override
+    public void onViewCreated(View view, Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        final TextView tv = (TextView) mRootView.findViewById(R.id.scroll_top);
+        tv.setText(mDataset[mPosition]);
+
+        mFromUnit = (CustomEditText) mRootView.findViewById(R.id.from_unit);
+        if (mPosition == 0) {
+            new Handler().postDelayed(new Runnable() {
+                @Override
+                public void run() {
+                    EventBus.post(new ConvertedValue(mFromUnit.getText().toString().trim(), mDataset[mPosition], mPosition, mSelectedConversion));
+                }
+            }, 2);
+        }
+
+        mFromUnit.addTextChangedListener(new TextWatcher() {
```

Comment:
@grishmashrestha check out textChangeListener in butterknife
http://jakewharton.github.io/butterknife/javadoc/